helm/chaos-mesh/templates/_helpers.tpl

```diff
@@ -31,6 +31,27 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/* Generate basic labels */}}
+{{- define "chaos-mesh.labels" -}}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
+app.kubernetes.io/part-of: {{ template "chaos-mesh.name" . }}
+app.kubernetes.io/version: {{ .Chart.AppVersion }}
+{{- if .Values.customLabels }}
+{{ toYaml .Values.customLabels }}
+{{- end }}
+{{- end }}
+
+{{/*
+Specify default selectors
+*/}}
+{{- define "chaos-mesh.selectors" }}
+app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```

Comment:
I think we could make the same change like "chaos-mesh.labels" for preventing extra whitespace. :D